onnxruntime/core/graph/model.cc

```diff
@@ -91,6 +91,10 @@ Model::Model(std::unique_ptr<ModelProto> model_proto, const IOnnxRuntimeOpSchema
         " specifies which version of the ONNX OperatorSet is being imported.");
   }
 
+  if (!model_proto->has_ir_version() || model_proto->ir_version() > kMaximumIrVersion) {
```

Comment:
instead of defining kMaximumIrVersion,
can't we use ONNX_NAMESPACE::Version::IR_VERSION ?

Reply:
just noticed this is not a PR to master. is that related?

Reply:
Brian M will eventually target `WindowsAI` into `master`. I mainly pointed this change to `WindowsAI` so our corresponding newly added test cases would pass (since WindowsAI uses the 'windowsai' branch as a submodule, rather than directly to `master`). I suppose it would have worked to put this change into master first and then cherry-pick it into WindowsAI...